lib/rules/multiline-ternary.js

```diff
@@ -18,10 +18,15 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-        schema: []
+        schema: [
+            {
+                enum: ["always", "never"]
+            },
```

Comment:
Looking at this example from https://spacetelescope.github.io/understanding-json-schema/reference/array.html:
```
{
  "type": "array",
  "items": [
    {
      "type": "number"
    },
    {
      "type": "string"
    },
    {
      "type": "string",
      "enum": ["Street", "Avenue", "Boulevard"]
    },
    {
      "type": "string",
      "enum": ["NW", "NE", "SW", "SE"]
    }
  ],
  "additionalItems": false
}
```
it looks like it's just for arrays, which this is not. It does look like we might want to do:
 ```
{
    "type": "string",
    "enum": ["always", "never"]
}
```

That being said, I think if we decide to do that we should make a separate issue and change that across the codebase.